src/Sylius/Bundle/AdminBundle/Resources/config/app/events.yaml

```
@@ -235,6 +235,30 @@ sylius_ui:
                     priority: 5
                     context:
                         event: sylius.admin.layout.topbar_right
+                        
+        sylius.admin.grid.filters:
+            blocks:
+                content:
+                    template: "@SyliusUi/Grid/Filter/_content.html.twig"
+                    priority: 10
+
+        sylius.admin.grid.content:
+            blocks:
+                content:
+                    template: "@SyliusUi/Grid/_content.html.twig"
+                    priority: 10
+
+        sylius.admin.grid.body:
+            blocks:
+                navigation:
+                    template: "@SyliusUi/Grid/_navigation.html.twig"
```

As these blocks are assigned to the one event, I would put them all in separate folder for ease of finding
```suggestion
                    template: "@SyliusUi/Grid/Body/_navigation.html.twig"
```